Tasks/AppCenterDistributeV3/task.json

```diff
@@ -52,6 +52,13 @@
             "required": true,
             "helpMarkDown": "Relative path from the repo root to the APK/AAB or IPA file you want to publish"
         },
+        {
+            "name": "buildVersion",
+            "type": "string",
+            "label": "Build version",
+            "required": false,
+            "helpMarkDown": "The free form string which needs to be specified for `*.zip` and `*.msi` binaries."
```

Comment:
Nit: more detail on what the string represents could be useful here

Reply:
@hagould In swagger definition it `The build version of the uploaded binary`, how do you find if we change help markdown to `The build version of the uploading binary which needs to be specified for `*.zip` and `*.msi payloads`?

Reply:
that sounds great!

Reply:
changed to :
The build version of the uploading binary which needs to be specified for `.zip` and `.msi` this value will be ignored unless the platform is WPF or WinForms